dash-pipeline/SAI/specs/sai_spec.yaml

```diff
@@ -10,6 +10,7 @@ api_types:
 - SAI_API_DASH_VNET
 - SAI_API_DASH_OUTBOUND_ROUTING
 - SAI_API_DASH_PA_VALIDATION
+- SAI_API_DASH_TUNNEL
```

Comment:
the SAI spec merge PR #577 is not merged, which causes the attributes to be inserted in the middle. Do you mind to help review #577, so we can avoid this to happen.

Reply:
Regenerated after merging latest master.